Hardware/Hardware/CardReader/UpdateTimeEstimate.swift

```diff
@@ -0,0 +1,18 @@
+/// The estimated amount of time for an update.
+/// Note that these times are estimates; actual times may vary depending on your network connection.
+public enum UpdateTimeEstimate {
```

Comment:
We are pretty much mapping the intervals that Stripe Terminal provides.

Reply:
This enum is tightly coupled to this SDK version's update time concept. I would almost rather we just used `TimeInterval` and had the convenience initializer`UpdateTimeEstimate+Stripe.swift` take Stripe's enum and use its upper bound to set the time interval. We could use 0 to map indeterminate.

Reply:
I am not sure I follow. How can we decouple this estimate from what the SDK declares? We can use different time segments (e.g. indeterminate, less than one minute, more than one minute, more than 15 minutes) but we still need to rely on the concept of time segments, so we might as well go with what the SDK provides?